config/locales/es.yml

```diff
@@ -0,0 +1,4 @@
+es:
+  logged_in: 'Has iniciado sesión'
+  logged_out: 'Cerrado sesión'
```

Comment:
Spanish native speaker here. I'd recommend to edit the following strings to:
* logged_out: `Su sesión ha finalizado correctamente` or `Cerrando sesión` (Although it sounds a bit weird)
* could_not_log_in: `No fue posible iniciar sesión en la tienda`